plugin/trino-cassandra/src/main/java/io/trino/plugin/cassandra/CassandraTypeMapping.java

```diff
@@ -0,0 +1,132 @@
+/*
```

Comment:
Please add another commit to rename `CassandraTypeMapping` to `CassandraType` as the 2nd commit.